bdsx/bds/scoreboard.ts

```diff
@@ -53,20 +61,36 @@ export class Scoreboard extends NativeClass {
         abstract();
     }
 
+    getScoreboardIdentityRef(id:ScoreboardId):ScoreboardIdentityRef {
+        abstract();
+    }
+
     removeObjective(objective:Objective):boolean {
         abstract();
     }
 
-    clearDisplayObjective(displaySlot:CxxString):Objective|null {
+    clearDisplayObjective(displaySlot:string):Objective|null {
         abstract();
     }
 
     setDisplayObjective(displaySlot:DisplaySlot, objective:Objective, order:ObjectiveSortOrder):DisplayObjective|null {
         abstract();
     }
+    resetPlayerScore(id:ScoreboardId, objective:Objective):void {
+        abstract();
+    }
+    // setPlayerScore(id:ScoreboardId, objective:Objective, value:number):number {
+    //     return this._modifyPlayerScore(false, id, objective, value, PlayerScoreSetFunction.Set);
+    // }
+    // addPlayerScore(id:ScoreboardId, objective:Objective, value:number):number {
+    //     return this._modifyPlayerScore(false, id, objective, value, PlayerScoreSetFunction.Add);
+    // }
+    // removePlayerScore(id:ScoreboardId, objective:Objective, value:number):number {
+    //     return this._modifyPlayerScore(false, id, objective, value, PlayerScoreSetFunction.Subtract);
+    // }
 }
 
-@nativeClass(null)
+@nativeClass()
```

Comment:
if the size is not null, it assumes the size from fields.
but there are not all fields of `Objective`.
Please set the size of the class instead or please add all fields or set it null.